topics/galaxy-interface/tutorials/collections/tutorial.md

```diff
@@ -106,19 +106,19 @@ By now we see that a collection can be used to bundle a large number of items in
 https://zenodo.org/record/5119008/files/chrM.fa.gz
 ```
 
-> ### {% icon details %} Set format to `fasta.gz`
+> ### {% icon hands_on %} Hands-on: Set format to `fasta.gz`
 > The above dataset is in `fasta.gz` format. The {% icon tip %} **Tip** section below explains how to upload these data and set the correct format. 
 >
 > {% snippet faqs/galaxy/datasets_import_via_link.md reset_form="True" link="https://zenodo.org/record/5119008/files/chrM.fa.gz" format="fasta.gz" %}
-{: .warning}
+{: .hands_on}
 
 ## Mapping reads
 
 **BWA-MEM** {% icon tool %} is a widely used sequence aligner for short-read sequencing datasets such as those we are analysing in this tutorial. (You can find the tool by typing `BWA MEM` in the search box at the top left corner of Galaxy interface).
 
 > ### {% icon hands_on %} Hands-on: Map sequencing reads to reference genome
 >
-> Run {% tool [BWA-MEM](toolshed.g2.bx.psu.edu/repos/devteam/bwa/bwa_mem/0.7.17.1) %} with the following parameters:
+> Run {% tool [BWA-MEM{% icon tool %}](toolshed.g2.bx.psu.edu/repos/devteam/bwa/bwa_mem/0.7.17.1) %} with the following parameters:
```

Comment:
```suggestion
> Run {% tool [BWA-MEM](toolshed.g2.bx.psu.edu/repos/devteam/bwa/bwa_mem/0.7.17.1) %} with the following parameters:
```

The tool icon here should be supplied automatically tbe the custom {% tool %} command